app/models/conversion_host.rb

```diff
@@ -454,7 +454,7 @@ def ansible_playbook(playbook, extra_vars = {}, miq_task_id = nil, auth_type = '
       raise MiqException::MiqInvalidCredentialsError, _("Unknown auth type: %{auth_type}") % {:auth_type => auth.authtype}
     end
 
-    params << {:extra_vars => "'#{extra_vars.to_json}'"}
+    extra_vars.each { |k, v| params << {:extra_vars= => "#{k}='#{v}'"} }
```

Comment:
It's ok to have multiple `:extra_vars`?

Reply:
@djberg96 yes, it's perfectly fine: https://docs.ansible.com/ansible/2.9/user_guide/playbooks_variables.html#passing-variables-on-the-command-line